### ISSUE TITLE
API Bug: CORS policy issue when making PUT requests

### ISSUE THREAD

**Issue by AndreasRiedel95** [freescout-helpdesk/freescout#3180]
**Description:**
I encountered a CORS policy error when attempting to make a PUT request using the Freescout API. The error message indicates that the PUT method is not allowed by the Access-Control-Allow-Methods in the preflight response. 

**Full error message:**
```
Access to fetch at 'HOSTED_FREESCOUT_URL' from origin 'URL' has been blocked by CORS policy: Method PUT is not allowed by Access-Control-Allow-Methods in preflight response.
```

**Steps to Reproduce:**
Make a PUT request to the Freescout API endpoint api/conversations/{conversationId}/custom_fields
Check the browser console or network tab for the CORS error message.

**Expected Behavior:**
I expect the PUT method to be allowed in the CORS policy and for the request to be processed successfully.

**Additional Information:**
The error occurs when making a PUT request to api/conversations/{conversationId}/custom_fields
I have confirmed that other HTTP methods like GET and POST work correctly.
I have already verified that the issue is not caused by my client-side code or browser configuration.

**Possible Solution:**
Based on the error message, it appears that the server's CORS policy needs to be modified to allow the PUT method. Adding the PUT method to the Access-Control-Allow-Methods header should resolve the issue.

PHP version: 8.2
Freescout version: 1.8.84

**Comment by freescout-helpdesk** [comment]
Do you have "Allowed CORS Hosts" specified in API & Webhooks settings in FreeScout?

**Comment by freescout-helpdesk** [comment]
Fixed in API &Webhooks Module v1.0.50
